### ISSUE TITLE
Official Docker images: missing -bin images

### ISSUE THREAD

**Issue by thePanz** [composer/docker#351]
Looks like the Docker registry does not have the `-bin` images anymore

See: https://hub.docker.com/_/composer/tags?name=2-bin

Those images are available on the GitHub repository tho

**Comment by alcohol** [comment]
The official repo never had the binary only tags. Since the binary cannot run without a php runtime, the binary only tags are not acceptable candidates for the official images repository.

**Comment by thePanz** [comment]
hum, I might get it wrong, but It did before :thinking: 
I am referring to https://github.com/composer/docker/pull/250
and a PR related to it: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6834

Probably I did not see the difference between `_/composer` (official) Vs. `composer/composer` (community) :thinking: 

Thanks for the update :+1:  and for maintaining this repository!

**Comment by alcohol** [comment]
Yeah they use `composer/composer` in that PR, which is the community image, not the official hub image.
